modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,49 +1,67 @@
 = AS2 Receive Endpoint Settings
 
-AS2 receive endpoint settings configure the source endpoint in an inbound message flow that receives B2B messages from partners directly or via third-party connections. One AS2 endpoint can receive data from multiple partners, third-party connections, or both.
+AS2 receive endpoint settings determine where to receive B2B messages received over AS2.
 
-Only a host can own AS2 receive endpoints. Use a host-owned AS2 receive endpoint for an inbound message flow that uses a third-party connection.
+An AS2 Receive from Partners endpoint (the only type of AS2 receive endpoint) acts as the source endpoint in an inbound message flow that receives B2B messages from partners either directly or via third-party connections.
 
-NOTE: Each AS2 receive endpoint configuration gets deployed as an individual Mule application in the runtime. Consider reusing an existing AS2 receive endpoint configuration across message flows.
+One AS2 endpoint can receive data from multiple partners, multiple third-party connections, or both.
+
+Only a host can own AS2 Receive from Partners endpoints.
 
-Create AS2 receive endpoints by configuring these settings on the Host page:
+Create an AS2 Receive from Partners endpoint by configuring these settings on the *<host>* page:
+
+NOTE: Each AS2 receive endpoint configuration gets deployed as an individual Mule application in the runtime. Consider reusing an existing AS2 receive endpoint configuration across message flows.
```

Comment:
If possible, try to move the NOTE to follow the table or use it as plain narrative introducing the topic, before Line 11. 
We want to avoid putting anything between an intro statement ending with `:` and the information pointed to by that `:`.